app/policies/upcoming/manage/facility_group_policy.rb

```diff
@@ -0,0 +1,27 @@
+class Upcoming::Manage::FacilityGroupPolicy < Upcoming::ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def allowed?
+    return true if user.super_admin?
+    user.accesses.admin.where(resource: resolve_record(record, FacilityGroup)).exists?
```

Comment:
This pattern is followed in all the policies for all managerial auth checks, notice that this is scoped by `admin`.